spec/components/home_page_hero_image_component_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe HomePageHeroImageComponent, type: :component do
+  let(:instance) { HomePageHeroImageComponent.new(override: override) }
+  let(:all_metadata) { instance.class.all_images_metadata }
+  let(:override) { nil }
+
+  describe "smoke test" do
+    it "renders" do
+      expect(render_inline(instance)).to have_selector('.hero-image')
+    end
+  end
+
+  describe "images in Rails.application.assets" do
+    it "all needed keys and images exist" do
+      expected_keys = ["link_title", "original_file", "path_1x", "path_2x", "work_friendlier_id"]
+      all_metadata.each do |metadata|
+        expected_keys.each do |k|
+          expect(metadata[k].class).to eq String
+          expect(Rails.application.assets.find_asset(metadata['path_1x']).class).to eq Sprockets::Asset
+          expect(Rails.application.assets.find_asset(metadata['path_2x']).class).to eq Sprockets::Asset
+        end
+      end
+    end
+  end
+
+  describe "tick = 0" do
+    before do
+      allow(instance).to receive(:tick).and_return(0)
+    end
+    it "renders placeholder" do
```

Comment:
What do you mean by the word "placeholder"?  I think of a placeholder as something used as a default when other content is unavailable. I don't think that's what this is testing for? Is this really "renders legend" or "renders title"? Or am I misunderstanding?